### PR TITLE
Fix an "strpos(): Empty needle" warning

### PR DESCRIPTION
**Pull request by toulet**
Signed-off-by: Cyrille TOULET <cyrille.toulet@linux.com>

### CODE REVIEW

File: system/core/URI.php [hunk 1]
```diff
@@ -205,11 +205,11 @@ protected function _parse_request_uri()
 		$query = isset($uri['query']) ? $uri['query'] : '';
 		$uri = isset($uri['path']) ? $uri['path'] : '';
 
-		if (strpos($uri, $_SERVER['SCRIPT_NAME']) === 0)
+		if (!empty($_SERVER['SCRIPT_NAME']) && strpos($uri, $_SERVER['SCRIPT_NAME']) === 0)
```
Comment (narfbg) [inline system/core/URI.php]: Needs to have a space on each side of `!`. Also, `empty()` would catch '0' or similar as well, which might be a really weird case, but still possible.

And if you'll be putting that in both conditions, it should really be just one if in which you'd nest these, so ...

    if (isset($_SERVER['SCRIPT_NAME'][0]))
    {
        // strpos ifs
    }

That would be the optimal solution.

**Comment by toulet** [comment]
This part of code has been updated.

**Comment by narfbg** [comment]
Now you're using 4 spaces for indentation. We use tabs ...

**Comment by toulet** [comment]
Oops, it's fixed ;~)

**Pull request merged**